### ISSUE TITLE
Make the repository link on asset page more prominent

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot-proposals#3638]
### Describe the project you are working on

I'm recently on an asset streak, submitting some stuff to AssetLib that I planned to publish eventually™.

### Describe the problem or limitation you are having in your project

I recently discovered the great possibilities that come with `.gitattributes` file, so I used it to create some cool GitHub pages for my assets. See https://github.com/KoBeWi/Godot-Palette-Swap-Shader for example.

The problem is that most of users won't see this page, because they will browse this asset in the AssetLib. I mention in the description that they can check repo for more info, but not sure if many people will do that if the repo link looks like this:
![image](https://user-images.githubusercontent.com/2223172/144762432-25e396e6-d8cc-4951-8883-066b6350e14f.png)
It's some random hard-to-spot "Show Files" button. I always even forget where it is.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be cool if the button was more like this:
![image](https://user-images.githubusercontent.com/2223172/144762508-ed63bb9f-4286-4ab3-8883-c55fb0161519.png)
Very easy to spot.

Obviously it wouldn't look like that, but the button should be *big* and clearly communicate that you can click it and it leads to useful things. And of course it should be optional, as a checkbox in the asset admin page. Not every asset has an useful repo, so there is no reason to display it for every asset. If someone makes a sophisticated README, they would be able to mark them asset as having such readme and this button would show up.

It's a bit more work for maintainers, because they'd need to confirm that the asset's repo is actually worth looking at. Or maybe it doesn't matter.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Add a checkbox when creating asset called e.g. "Display Repository Button". Asset with this enabled will have a big button on its AssetLib page.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Not really. You can advertise your repo in the description or in the screenshot, but it's a bit useless if the actual repo link is hard to discover.

### Is there a reason why this should be core and not an add-on in the asset library?

It's about asset library. Having it as a plugin would be sooo meta 🤔

**Comment by Calinou** [comment]
> Obviously it wouldn't look like that, but the button should be big and clearly communicate that you can click it and it leads to useful things. And of course it should be optional, as a checkbox in the asset admin page. Not every asset has an useful repo, so there is no reason to display it for every asset. If someone makes a sophisticated README, they would be able to mark them asset as having such readme and this button would show up.

I don't think this button should be optional. I would just opt for making it a standard Button visible on all asset detail pages, and refactor the asset detail page to look more like the web frontend.

**Comment by KoBeWi** [comment]
It just occurred to me 🤔
If Godot could support Markdown, asset lib could just have an option to display README as detail page. To prevent uncontrolled modifications, you'd have to provide exact commit hash from which the README would be used.

But aside from markdown, editor would need to support things like GIFs etc. to make sure all pages are displayed correctly.

**Comment by KoBeWi** [comment]
The new Asset Store integration resolved this.
